2013/05/19 18:00:03 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE 'tbl_user' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"username" varchar(255) NOT NULL,
	"email" varchar(255) NOT NULL,
	"password" varchar(255) NOT NULL,
	"last_login_time" datetime DEFAULT NULL,
	"create_time" datetime DEFAULT NULL,
	"create_user_id" int(11) DEFAULT NULL,
	"update_time" datetime DEFAULT NULL,
	"update_user_id" int(11) DEFAULT NULL
) ENGINE=InnoDB
2013/05/19 18:00:03 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 table 'tbl_user' already exists. The SQL statement executed was: CREATE TABLE 'tbl_user' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"username" varchar(255) NOT NULL,
	"email" varchar(255) NOT NULL,
	"password" varchar(255) NOT NULL,
	"last_login_time" datetime DEFAULT NULL,
	"create_time" datetime DEFAULT NULL,
	"create_user_id" int(11) DEFAULT NULL,
	"update_time" datetime DEFAULT NULL,
	"update_user_id" int(11) DEFAULT NULL
) ENGINE=InnoDB.
2013/05/19 18:00:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 table 'tbl_user' already exists. The SQL statement executed was: CREATE TABLE 'tbl_user' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"username" varchar(255) NOT NULL,
	"email" varchar(255) NOT NULL,
	"password" varchar(255) NOT NULL,
	"last_login_time" datetime DEFAULT NULL,
	"create_time" datetime DEFAULT NULL,
	"create_user_id" int(11) DEFAULT NULL,
	"update_time" datetime DEFAULT NULL,
	"update_user_id" int(11) DEFAULT NULL
) ENGINE=InnoDB' in C:\Users\rocho\work\xampp\htdocs\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\rocho\work\xampp\htdocs\yii\framework\db\CDbCommand.php(1321): CDbCommand->execute()
#1 C:\Users\rocho\work\xampp\htdocs\yii\framework\db\CDbMigration.php(228): CDbCommand->createTable('tbl_user', Array, 'ENGINE=InnoDB')
#2 C:\Users\rocho\work\xampp\htdocs\auth\protected\migrations\m130519_155432_create_user_table.php(18): CDbMigration->createTable('tbl_user', Array, 'ENGINE=InnoDB')
#3 C:\Users\rocho\work\xampp\htdocs\yii\framework\cli\commands\MigrateCommand.php(385): m130519_155432_create_user_table->up()
#4 C:\Users\rocho\work\xampp\htdocs\yii\framework\cli\commands\MigrateCommand.php(109): MigrateCommand->migrateUp('m130519_155432_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 C:\Users\rocho\work\xampp\htdocs\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 C:\Users\rocho\work\xampp\htdocs\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#8 C:\Users\rocho\work\xampp\htdocs\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#9 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#10 C:\Users\rocho\work\xampp\htdocs\yii\framework\yiic.php(33): CApplication->run()
#11 C:\Users\rocho\work\xampp\htdocs\auth\protected\yiic.php(7): require_once('C:\Users\rocho\...')
#12 {main}
---
2013/05/19 18:02:05 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_auth.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
2013/05/20 16:30:45 [error] [php] include(password_repeat.php): failed to open stream: No such file or directory (C:\Users\rocho\work\xampp\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\Users\rocho\work\xampp\htdocs\yii\framework\validators\CValidator.php(186): import()
#1 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CModel.php(286): createValidator()
#2 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CModel.php(260): User->createValidators()
#3 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CModel.php(303): User->getValidators()
#4 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1237): User->isAttributeRequired()
#5 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(562): activeLabelEx()
#6 C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\views\user\_form.php(21): CActiveForm->labelEx()
#7 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#8 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CBaseController.php(95): UserController->renderInternal()
#9 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(869): UserController->renderFile()
#10 C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\views\user\create.php(18): UserController->renderPartial()
#11 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#12 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CBaseController.php(95): UserController->renderInternal()
#13 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(869): UserController->renderFile()
#14 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(782): UserController->renderPartial()
#15 C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\controllers\UserController.php(79): UserController->render()
#16 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#17 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#18 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): UserController->runAction()
#19 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#21 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): UserController->filterAccessControl()
#22 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#23 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#24 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(265): UserController->runActionWithFilters()
#25 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CWebApplication.php(282): UserController->run()
#26 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#27 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#28 C:\Users\rocho\work\xampp\htdocs\auth\index.php(13): CWebApplication->run()
REQUEST_URI=/auth/index.php?r=auth/user/create
in C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\views\user\_form.php (21)
in C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\views\user\create.php (18)
in C:\Users\rocho\work\xampp\htdocs\auth\protected\modules\auth\controllers\UserController.php (79)
2013/05/20 16:52:48 [error] [php] include(User.php): failed to open stream: No such file or directory (C:\Users\rocho\work\xampp\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\Users\rocho\work\xampp\htdocs\auth\protected\components\UserIdentity.php(13): spl_autoload_call()
#1 C:\Users\rocho\work\xampp\htdocs\auth\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\Users\rocho\work\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\Users\rocho\work\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#4 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\Users\rocho\work\xampp\htdocs\auth\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\Users\rocho\work\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\Users\rocho\work\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\Users\rocho\work\xampp\htdocs\auth\index.php(13): CWebApplication->run()
REQUEST_URI=/auth/index.php?r=site/login
in C:\Users\rocho\work\xampp\htdocs\auth\protected\components\UserIdentity.php (13)
in C:\Users\rocho\work\xampp\htdocs\auth\protected\models\LoginForm.php (52)
in C:\Users\rocho\work\xampp\htdocs\auth\protected\controllers\SiteController.php (94)
2013/08/30 20:31:58 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_konferencia.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
